libraries/windows_helper.rb

```diff
@@ -120,6 +123,35 @@ def to_array(var)
       var.reject(&:nil?)
     end
 
+    def openssl_cert_obj
+      OpenSSL::X509::Certificate.new(raw_source)
+    end
+
+    def add_cert(cert_obj)
+      store = Win32::Certstore.open(store_name)
+      store.add(cert_obj)
+    end
+
+    def delete_cert
```

Comment:
We should just move these new helper methods right into the action class of the resource. That's the way we'll use it in chef-client itself if/when we move this resource into chef and it prevents doing requires on every chef node when we don't actually use these libraries. This will save